LabeledIcon/LabeledIcon.js

```diff
@@ -11,7 +11,7 @@
  */
 
 import kind from '@enact/core/kind';
-import UiLabeledIcon from '@enact/ui/LabeledIcon';
+import {LabeledIconBase as UiLabeledIcon} from '@enact/ui/LabeledIcon';
```

Comment:
Let's rename `LabeledIconBase` as `UiLabeledIconBase` and wrap `agate/LabeledIconBase` with `ui/LabeledIconDecorator` like `agate/Button`.

```
import {LabeledIconBase as UiLabeledIconBase, LabeledIconDecorator as UiLabeledIconDecorator} from '@enact/ui/LabeledIcon';

...

 * @hoc
 * @memberof agate/LabeledIcon
 * @mixes agate/LabeledIcon.UiLabeledIconDecorator
 * @mixes agate/Skinnable.Skinnable
 * @mixes ui/Pure.Pure
 * @public
 */
const LabeledIconDecorator = compose(
	UiLabeledIconDecorator,
	Pure,
	Skinnable
);
```